test/MusicStore.Test/CheckoutControllerTest.cs

```diff
@@ -43,6 +48,66 @@ public void AddressAndPayment_ReturnsDefaultView()
         }
 
         [Fact]
+        public async Task AddressAndPayment_RedirectToCompleteWhenSuccessful()
+        {
+            // Arrange
+            var httpContext = new DefaultHttpContext();
+
+            var orderId = 10;
+            var order = new Order()
+            {
+                OrderId = orderId,
```

Comment:
In another review, I was told that leaving , is better since we don't have to touch the line in case we have to add another line. I think it could be a personal preference. I'd like to leave , here for now. :-)